huaweicloud/services/acceptance/dataarts/resource_huaweicloud_dataarts_studio_business_metric_test.go

```diff
@@ -0,0 +1,257 @@
+resource "huaweicloud_dataarts_studio_business_metric" "test" {
+  name             = "%[1]s"
+  workspace_id     = "%[2]s"
+  biz_catalog_id   = "%[3]s"
+  owner            = "test-owner"
+  owner_department = "test-department"
+  time_filters     = "双周"
+  interval_type    = "HOUR"
+  destination      = "test destination"
+  definition       = "test definition"
+  expression       = "a+b+c"
+  description      = "test description"
+  apply_scenario   = "test apply scenario"
+  technical_metric = 100
+  measure          = "test measure"
+  general_filters  = "test general filters"
+  data_origin      = "test data origin"
+  unit             = "test unit"
+  name_alias       = "alias-name"
+  code             = "testCode"
+}
+`, name, acceptance.HW_DATAARTS_WORKSPACE_ID, acceptance.HW_DATAARTS_BIZ_CATALOG_ID)
+}
+
+func testBusinessMetric_basic_update1(name string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_dataarts_studio_business_metric" "test" {
+  name             = "%[1]s"
+  workspace_id     = "%[2]s"
+  biz_catalog_id   = "%[3]s"
+  owner            = "test-owner-update"
+  owner_department = "test-department-update"
+  time_filters     = "周,季度,其他"
+  interval_type    = "WEEK"
+  destination      = "test destination update"
+  definition       = "test definition update"
+  expression       = "a+b+c+d"
+  description      = "test description update"
+  apply_scenario   = "test apply scenario update"
+  technical_metric = 200
+  measure          = "test measure update"
+  general_filters  = "test general filters update"
+  data_origin      = "test data origin update"
+  unit             = "test unit update"
+  name_alias       = "alias-name-update"
+  code             = "testCodeUpdate"
+}
+`, name, acceptance.HW_DATAARTS_WORKSPACE_ID, acceptance.HW_DATAARTS_BIZ_CATALOG_ID)
+}
+
+func testBusinessMetric_basic_update2(name string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_dataarts_studio_business_metric" "test" {
+  name             = "%[1]s"
+  workspace_id     = "%[2]s"
+  biz_catalog_id   = "%[3]s"
+  owner            = "test-owner-update"
+  owner_department = "test-department-update"
+  time_filters     = "周,季度,其他"
+  interval_type    = "WEEK"
+  destination      = "test destination update"
+  definition       = "test definition update"
+  expression       = "a+b+c+d"
+  code             = "testCodeUpdate"
+}
+`, name, acceptance.HW_DATAARTS_WORKSPACE_ID, acceptance.HW_DATAARTS_BIZ_CATALOG_ID)
+}
+
+// testDataArtsStudioImportState use to return an id with format <workspace_id>/<id>
+func testDataArtsStudioImportState(name string) resource.ImportStateIdFunc {
+	return func(s *terraform.State) (string, error) {
+		rs, ok := s.RootModule().Resources[name]
+		if !ok {
+			return "", fmt.Errorf("resource (%s) not found: %s", name, rs)
+		}
+
+		workspaceID := rs.Primary.Attributes["workspace_id"]
+		if workspaceID == "" {
+			return "", fmt.Errorf("attribute (workspace_id) of Resource (%s) not found: %s", name, rs)
```

Comment:
`rs` is not string type
```suggestion
			return "", fmt.Errorf("attribute (workspace_id) of Resource (%s) not found", name)
```

Reply:
OK